pimd/pim6_cmd.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * PIM for IPv6 FRR
+ * Copyright (C) 2022  Mobashshera Rasool <mrasool@vmware.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; see the file COPYING; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+#ifndef PIM6_CMD_H
+#define PIM6_CMD_H
+#define PIM_STR                                "PIM information\n"
+#define GM_STR                                 "IGMP or MLD information\n"
```

Comment:
Since we are going to use separate pim6_cmd.h file, lets add just this "MLD information\n" here. IGMP is not required. Also rename is to MLD_STR since this macro is only used in pim6_cmd.h and not anywhere else.

Reply:
Fixed